### PR TITLE
Ux improvements

### PR DESCRIPTION
**Pull request by jonedollarz**

### CODE REVIEW

File: src/tile-link/_tile-link-with-background.scss [hunk 1]
```diff
@@ -20,3 +18,40 @@
...
+    @extend .polaris-bg-default;
+  }
+
+  .tile-bg-active {
+    @include transition(opacity $med-transition ease);
+    opacity: 0;
+    @extend .polaris-bg-sky-offers-dark;
+  }
+
+  &:hover{
```
Comment (david-nicholson) [inline src/tile-link/_tile-link-with-background.scss]: space before the <code>{</code>!!!

File: demo/styles/tile.css [hunk 2]
```diff
@@ -970,3 +992,26 @@
...
 .tile.theme-channel .tile-link:hover .tile-title {
   color: #222222; }
+
+.tile.theme-text-with-background .tile-bg {
+  position: absolute;
+  top: 50%;
+  right: 0;
+  bottom: 0;
+  left: 0; }
+  .tile.theme-text-with-background .tile-bg > div {
```
Comment (aswitalski) [inline demo/styles/tile.css:1002]: Could we use a class instead of a div?

File: demo/styles/tile.css [hunk 3]
```diff
@@ -172,6 +172,8 @@
   color: white;
```
Comment (aswitalski) [inline demo/styles/tile.css:172]: Even if this file is generated could we format the code?

File: src/tile-link/_tile-link-with-background.scss [hunk 4]
```diff
@@ -20,3 +18,40 @@
...
+.tile.theme-text-with-background {
+
+  .tile-bg {
+    position: absolute;
+    top: 50%;
+    right: 0;
+    bottom: 0;
+    left: 0;
+
+    > div {
```
Comment (aswitalski) [inline src/tile-link/_tile-link-with-background.scss:31]: Can we change this to a class as well?

File: demo/styles/tile.css [hunk 5]
```diff
@@ -205,13 +207,18 @@
...
   background-color: #FFFFFF;
   background-image: -webkit-radial-gradient(#FFFFFF, #E1E0E4);
   background-image: radial-gradient(#FFFFFF, #E1E0E4); }
 
+.polaris-bg-default-dark, .tile.theme-text-with-background .tile-bg-active {
+  background: url(data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiA/Pgo8c3ZnIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgd2lkdGg9IjEwMCUiIGhlaWdodD0iMTAwJSIgdmlld0JveD0iMCAwIDEgMSIgcHJlc2VydmVBc3BlY3RSYXRpbz0ibm9uZSI+CiAgPHJhZGlhbEdyYWRpZW50IGlkPSJncmFkLXVjZ2ctZ2VuZXJhdGVkIiBncmFkaWVudFVuaXRzPSJ1c2VyU3BhY2VPblVzZSIgY3g9IjUwJSIgY3k9IjUwJSIgcj0iNzUlIj4KICAgIDxzdG9wIG9mZnNldD0iMCUiIHN0b3AtY29sb3I9IiNmZmZmZmYiIHN0b3Atb3BhY2l0eT0iMSIvPgogICAgPHN0b3Agb2Zmc2V0PSIxMDAlIiBzdG9wLWNvbG9yPSIjYjNiM2IzIiBzdG9wLW9wYWNpdHk9IjEiLz4KICA8L3JhZGlhbEdyYWRpZW50PgogIDxyZWN0IHg9Ii01MCIgeT0iLTUwIiB3aWR0aD0iMTAxIiBoZWlnaHQ9IjEwMSIgZmlsbD0idXJsKCNncmFkLXVjZ2ctZ2VuZXJhdGVkKSIgLz4KPC9zdmc+);
+  background-color: #B3B3B3;
+  background-image: -webkit-radial-gradient(#FFFFFF, #B3B3B3);
```
Comment (aswitalski) [inline demo/styles/tile.css:219]: Instead of prefixing we can use a mixin...

**Comment by jonedollarz** [comment]
this selector is used in many places through out the project

File: demo/styles/tile.css (hunk 3 above)
Comment (david-nicholson) [inline demo/styles/tile.css:172]: Why? It's just for an example page and compiled through node-sass. Should arguably be minified rather than formatted...

Comment (aswitalski) [inline demo/styles/tile.css:172]: We could add that to the build, but not absolutely needed.

File: src/tile-link/_tile-link-with-background.scss (hunk 4 above)
Comment (david-nicholson) [inline src/tile-link/_tile-link-with-background.scss:31]: Agree. It would be good to get rid of the generic <code>div</code>

**Comment by aswitalski** [comment]
I would still change the selectors, but if this is just a replication of the already-existing solution :shipit:

File: demo/styles/tile.css (hunk 5 above)
Comment (david-nicholson) [inline demo/styles/tile.css:219]: I believe that's brought in from the gradients project and will be generated there. Therefore @jonedollarz isn't prefixing.

**Comment by david-nicholson** [comment]
I think it's a fair point to change the <code>> div</code>. Should be really quick and would be an improvement to refactor while we're on it?

**Pull request merged**